app/src/main/java/chat/rocket/android/renderer/MessageRenderer.java

```diff
@@ -76,10 +75,11 @@ public MessageRenderer timestampInto(TextView textView) {
     }
 
     switch (object.getSyncState()) {
```

Comment:
Shouldn't it be better if we put this way:

case SyncState.FAILED:
         textView.setText(R.string.failed_to_sync);
         break;